cde/src/chipsalliance/rocketchip/config.scala

```diff
@@ -168,19 +172,23 @@ object config {
     def find[T](pname: Field[T]): Option[T] = pname.default
   }
 
-  private class ChainView(head: Parameters, site: View, up: View) extends View {
```

Comment:
@sequencer I believe the original behavior was correct.
`site` refers to the parameters at the "site", where the call chain started. It should be passed along unmodified recursively. 
This is why the WithRV32 thing worked, since `site(XLen)` was like a global field (it doesn't make sense to have XLen return different values in different places.)

Reply:
Indeed. I'm highly suspicious of this change. Did our fork have the same change?

Reply:
No